frontend/components/SetupPage/Create/SetupEoaFunding.tsx

```diff
@@ -80,7 +88,7 @@ const SetupEoaFundingWaiting = ({ chainName }: SetupEoaFundingWaitingProps) => {
         </Flex>
 
         <span className="can-select-text break-word">
-          {`ETH: ${masterEoaAddress}`}
+          {`ETH: ${masterEoaAddress || NA}`}
```

Comment:
will pull `homeChainId` from `selectedAgentConfig` in `useServices` in future, then `getNativeTokenSymbol(homeChainId)` to get native gas token

not needed for now, but can hardcode XDAI instead for now

```suggestion
          {`XDAI: ${masterEoaAddress || NA}`}
```